### PR TITLE
HYDRA-1161 : Add a unit test for selection highlight of usd instancea…

### PR DESCRIPTION
**Pull request by lanierd-adsk**
…ble prims

Disable the testMaterialXOnNative unit test until get get a build of lookdevX.

### CODE REVIEW

**Comment by lanierd-adsk** [comment]
pr-build

**Comment by lanierd-adsk** [comment]
The JIRA to update the lookdevX build is HYDRA-1320 and there is alink explaining the the unit test matyerialXOnNative is disabled in the current JIRA/PR HYDRA-1161

File: test/testSamples/testStageInstanceablePrimsSelHighlight/testInstanceablePrimsSelHighlight.ma [hunk 1]
```diff
@@ -0,0 +1,239 @@
+//Maya ASCII 2026ff02 scene
```
Comment (lilike-adsk) [inline test/testSamples/testStageInstanceablePrimsSelHighlight/testInstanceablePrimsSelHighlight.ma]: This needs to be changed to 2025XXXX

File: test/testSamples/testStageInstanceablePrimsSelHighlight/testInstanceablePrimsSelHighlight.ma [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+//Name: testInstanceablePrimsSelHighlight.ma
+//Last modified: Fri, Dec 06, 2024 11:34:05 AM
+//Codeset: 1252
+requires maya "2026ff02";
```
Comment (lilike-adsk) [inline test/testSamples/testStageInstanceablePrimsSelHighlight/testInstanceablePrimsSelHighlight.ma]: same here

File: test/testSamples/testStageInstanceablePrimsSelHighlight/testInstanceablePrimsSelHighlight.ma [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+requires -nodeType "mayaUsdLayerManager" -nodeType "mayaUsdProxyShape" -dataType "pxrUsdStageData"
+		 "mayaUsdPlugin" "0.31.0";
+currentUnit -l centimeter -a degree -t film;
+fileInfo "application" "maya";
+fileInfo "product" "Maya 2026";
```
Comment (lilike-adsk) [inline test/testSamples/testStageInstanceablePrimsSelHighlight/testInstanceablePrimsSelHighlight.ma]: and here

**Comment by lanierd-adsk** [comment]
build-pr

**Comment by lanierd-adsk** [comment]
pr-build

**Review by lilike-adsk — APPROVED** [review]

**Pull request merged**